widget/embedded/src/utils/wallets.ts

```diff
@@ -82,7 +88,11 @@ export function mapWalletTypesToWalletInfo(
         img: image,
         installLink,
         showOnMobile,
+        supportedChains,
       } = getWalletInfo(type);
+      const types = supportedChains.map((item) => item.type);
+      const blockchainTypes = [...new Set(types)];
```

Comment:
I believe we have a helper function called `removeDuplicateFrom(array)`. You can utilize it in this context.